docs/installing-solidity.rst

```diff
@@ -300,6 +300,27 @@ CMake options
 
 If you are interested what CMake options are available run ``cmake .. -LH``.
 
+SMT Solvers
+-----------
+In some cases, installation may fail due to issues related to the SMT solvers
```

Comment:
I'm not sure about this text - maybe we should just state, that solidity can be built against SMT solvers and will do so by default, if they are found in the system, but that each solver can be disabled by a cmake option, instead of focusing on build failures? We might still want to mention that this can be a workaround for potential build failures, but maybe only as a side note? (hopefully all of these build failures will soon be fixed anyways ;))

But I'd be fine with it as it is as well. (@axic, what do you think?)

Reply:
I think your text makes sense.